schema.graphql

```diff
@@ -246,6 +246,22 @@ input BigIntSearchOptions {
   lte: BigInt
 }
 
+type Blueprint {
+  admins: [User!]!
+  created_at: String!
+  form_values: JSON!
+  id: Float!
+  minted: Boolean!
+  minter_address: String!
+}
+
+input BlueprintWhereInput {
+  admin_address: StringSearchOptions
```

Comment:
Similar question as for collections, there's only one admin possible?

Reply:
Resolved in https://github.com/hypercerts-org/hypercerts-api/pull/177/commits/194cb7330f09d8396b50450b7bfcb8734b104212